docs/tutorials/intro.md

```diff
@@ -6,39 +6,44 @@ The library provides a layer of APIs for all software to enable internationaliza
 
 To use `ICU4X` in the Rust ecosystem one can either add dependencies on selected components, or add a dependency on the meta-crate `icu` which brings a reasonable selection of components in the most user-friendly configuration of features.
 
-In this tutorial we are going to start with the meta-crate and then introduce a customization.
+In this tutorial we are going to build up to writing an app that uses the `icu::datetime` component to format a date and time in a Japanese locale, covering various topics in the process.
 
 # 1. Requirements
 
-For this tutorial we assume the user has basic Rust knowledge. If acquiring it is necessary, [Rust Book](https://doc.rust-lang.org/book/) provides an excellent introduction.
+For this tutorial we assume the user has basic Rust knowledge. If acquiring it is necessary, the [Rust Book](https://doc.rust-lang.org/book/) provides an excellent introduction.
 We also assume that the user is familiar with a terminal and have `git`, `rust` and `cargo` installed.
 
 To verify that, open a terminal and check that the results are similar to:
 
 ```
-user@host:~/projects/icu$ git --version
+user@host:~/projects/icu_tutorial$ git --version
 git version 2.31.1
-user@host:~/projects/icu$ cargo --version
+user@host:~/projects/icu_tutorial$ cargo --version
 cargo 1.51.0 (43b129a20 2021-03-16)
 ```
 
-In this tutorial we are going to use a directory relative to the user's home directory `~/projects/icu/`. The `~` in the path indicates the relative location of the user home directory.
-Please create the directory structure necessary.
+In this tutorial we are going to use a directory relative to the user's home directory `~/projects/icu_tutorial/`. The `~` in the path indicates the relative location of the user home directory.
 
 # 2. Creating MyApp
 
 To initialize a binary application, navigate to the root directory of our project and initialize a new binary app called `myapp`:
 
 ```
-cd ~/projects/icu
-cargo init --bin myapp
+cd ~/projects/icu_tutorial
+cargo new --bin myapp
 ```
 
-The result is a new directory `~/projects/icu/myapp` with a file `./src/main.rs` which is the main file for our application.
+The result is a new directory `~/projects/icu_tutorial/myapp` with a file `./src/main.rs` which is the main file for our application.
```

Comment:
If we're assuming `cargo` knowledge, this is redundant. If not, we should use relative paths.

```suggestion
The result is a new directory `myapp` with a file `src/main.rs` which is the main file for our application.
```